common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesAccountCredentialCache.java

```diff
@@ -422,7 +424,10 @@ public static CredentialType getCredentialTypeForCredentialCacheKey(@NonNull fin
             if (cacheKey.contains(CACHE_VALUE_SEPARATOR + credentialTypeStr + CACHE_VALUE_SEPARATOR)) {
                 Logger.verbose(TAG, "Cache key is a Credential type...");
 
-                if (credentialTypeStr.equalsIgnoreCase(CredentialType.AccessToken.name())) {
+                // We treat AccessToken & AccessToken_With_Authscheme as the same type
```

Comment:
Personally, I'd prefer making this explicit - we could have two if clauses that returns the same type (AccessTokenRecord.class)

Reply:
+1

Reply:
Resolved in 7f5e41c1f597651672b79d9f3ee80e7b3729b0eb